### PR TITLE
tests(plugins/zipkin) make `host` and `port` of `Zipkin` configurable

### PR DESCRIPTION
**Pull request by ADD-SP**
### Summary

Now, the `host` and `port` for `Zipkin` are fixed in the code, which is not conducive to developers running tests locally. These two parameters should be allowed to be changed via environment variables.

This PR introduces two environment variables to improve it:

* `KONG_SPEC_TEST_ZIPKIN_HOST`
* `KONG_SPEC_TEST_ZIPKIN_PORT`

### CODE REVIEW

**Review by Suika-Kong — APPROVED** [review]

File: spec/helpers.lua [hunk 1]
```diff
@@ -20,6 +20,8 @@ local MOCK_UPSTREAM_SSL_PORT = 15556
 local MOCK_UPSTREAM_STREAM_PORT = 15557
 local MOCK_UPSTREAM_STREAM_SSL_PORT = 15558
 local MOCK_GRPC_UPSTREAM_PROTO_PATH = "./spec/fixtures/grpc/hello.proto"
+local ZIPKIN_HOST = os.getenv("KONG_SPEC_TEST_ZIPKIN_HOST") or "localhost"
+local ZIPKIN_PORT = tonumber(os.getenv("KONG_SPEC_TEST_ZIPKIN_PORT") or 9411)
```
Comment (Suika-Kong) [inline spec/helpers.lua:24]: not really important but

```suggestion
local ZIPKIN_PORT = tonumber(os.getenv("KONG_SPEC_TEST_ZIPKIN_PORT")) or 9411
```

**Comment by Tieske** [comment]
just see they are the same as the Redis ones, so that should be fine.

**Review by Tieske — CHANGES_REQUESTED** [review]
the documenta comments need to be updated as well; https://github.com/Kong/kong/blob/30af56b548bf1bee6a8c493025b5498291de4afc/spec/helpers.lua#L2784

File: spec/helpers.lua [hunk 2]
```diff
@@ -2797,7 +2799,10 @@ end
 -- @field mock_upstream_stream_port
 -- @field mock_upstream_stream_ssl_port
 -- @field mock_grpc_upstream_proto_path
--- @field redis_host The hostname for a Redis instance if available. Port should be `6379`.
+-- @field redis_host The host for Redis
+-- @field redis_port The port (SSL disabled) for Redis
+-- @field redis_ssl_port The port (SSL enabled) for Redis
+-- @field redis_ssl_sni The server name for Redis
```
Comment (Tieske) [inline spec/helpers.lua:2805]: good to add the Redis ones, but the Zipkin ones are still missing?

File: spec/helpers.lua [hunk 3]
```diff
@@ -2799,12 +2799,12 @@ end
...
 -- @field mock_grpc_upstream_proto_path
--- @field redis_host The host for Redis
--- @field redis_port The port (SSL disabled) for Redis
--- @field redis_ssl_port The port (SSL enabled) for Redis
--- @field redis_ssl_sni The server name for Redis
--- @field zipkin_host The host for Zipkin service
--- @field zipkin_port the port for Zipkin service
+-- @field redis_host The host for Redis, it can be set by env KONG_SPEC_TEST_REDIS_HOST.
+-- @field redis_port The port (SSL disabled) for Redis, it can be set by env KONG_SPEC_TEST_REDIS_PORT.
+-- @field redis_ssl_port The port (SSL enabled) for Redis, it can be set by env KONG_SPEC_TEST_REDIKONG_SPEC_TEST_REDIS_SSL_PORTS_PORT.
```
Comment (Tieske) [inline spec/helpers.lua:2804]: this is a typo. Can you fix?

**Pull request merged**